packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-point-view.tsx

```diff
@@ -32,22 +30,30 @@ const hitboxSizePx = 48;
 // on an interactive graph.
 export const MovablePointView = forwardRef(
     (props: Props, hitboxRef: ForwardedRef<SVGGElement>) => {
-        const {markings, showTooltips, interactiveColor} = useGraphConfig();
+        const {
+            markings,
+            showTooltips,
+            interactiveColor,
+            disableKeyboardInteraction,
+        } = useGraphConfig();
         const {
             point,
-            color = interactiveColor,
             dragging,
             focused,
             cursor,
             showFocusRing,
             onClick = () => {},
         } = props;
 
-        // WB Tooltip requires a color name for the background color.
-        // Since the color in props is a hex value, a reverse lookup is needed.
-        const wbColorName = (Object.entries(WBColor).find(
-            ([_, value]) => value === color,
-        )?.[0] ?? "blue") as keyof typeof WBColor;
```

Comment:
Since MovablePoint is never any color except blue or gray, I removed this backwards search and made it more explicit.